src/helpers/default.js

```diff
@@ -1,3 +1,11 @@
 /* eslint-disable import/prefer-default-export */
 // eslint-disable-next-line no-useless-escape
 export const regexForEmail = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+export const roleIds = {
+  'Super Administrator': 1,
```

Comment:
I remember mentioning that you should change this to `camelCase` without the string.

Reply:
My apologies for this. It has been fixed